rss.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/mmcdole/gofeed"
+	ext "github.com/mmcdole/gofeed/extensions"
+)
+
+type Feed struct {
+		Username    string
+		DisplayName string
+		IconURL     string
+		Entries     []*FeedEntry
+}
+
+type FeedEntry struct {
+		ID          string
+		URL         string
+		Title       string
+		Year        string
+		Rating      int
+		WatchedDate time.Time
+		Rewatch     bool
+		Poster      string
+		Review      string
+}
+
+// Fetches a user's RSS feed, returning an array of 50 FeedEntrys with parsed values
+func GetFeed(username string) (Feed) {
+	fp := gofeed.NewParser()
+	feed, err := fp.ParseURL("https://letterboxd.com/" + username + "/rss/")
+
+	if err != nil {
+		log.Fatalf("failed to fetch feed: %v\n", err)
```

Comment:
Don't use `Fatalf`, just return error and let that function handle things. We don't want the whole bot to crash just because it failed to get an rss feed.